package.json

```diff
@@ -14,13 +14,16 @@
   "devDependencies": {
```

Comment:
It should work if you start the test with `npm test`. It won't work if you start the test via `gulp`.